src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/DeliveryRequestResolver.java

```diff
@@ -90,13 +92,12 @@ public boolean canResolve(
             return null;
         }
 
-        JobDeliveryman job = (JobDeliveryman) freeDeliveryMan.getJob();
+        final JobDeliveryman job = (JobDeliveryman) freeDeliveryMan.getJob();
         job.addRequest(request.getToken());
 
         return Lists.newArrayList();
     }
 
-    @Nullable
     @Override
     public void resolve(
                          @NotNull final IRequestManager manager, @NotNull final IRequest<? extends Delivery> request) throws RuntimeException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.RuntimeException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)